recipes/librdkafka/all/conanfile.py

```diff
@@ -21,6 +21,7 @@ class LibrdkafkaConan(ConanFile):
         "ssl": [True, False],
         "sasl": [True, False],
         "lz4": [True, False],
+        "refcnt_debug": [True, False],
```

Comment:
Should this option even exist?
Isn't this just a ref counting self test of the librdkafka library?
How about just disabling it always?

Reply:
That's a good question.

Reply:
@intelligide please, clarify.

Reply:
So after doing some digging into this I can confirm what @madebr pointed out.

Prints reference counters: https://github.com/edenhill/librdkafka/commit/21dc38ddf60d50fdfd314efca54177a1a90198b7

It's a part of the CI of the project to spot check memory leaks: https://github.com/edenhill/librdkafka/commit/5c27746252e81e436750749a6672b68ca4b68dd4

```suggestion
```